tests/bdd/scale/test_NAS_T1067.py

```diff
@@ -54,6 +54,8 @@ def you_should_be_on_the_dashboard_click_on_the_accounts_on_the_side_menu_and_cl
 @when('the Users page should open, click on the "Add" Button')
 def the_users_page_should_open_click_on_the_add_button(driver):
     """the Users page should open, click on the "Add" Button."""
+    assert wait_on_element(driver, 7, '//h1[contains(.,"Display Note")]')
+    driver.find_element_by_xpath('//button[@ix-auto="button__CLOSE"]').click()  
```

Comment:
at the speed that the test is running you should add wait for '//button[@ix-auto="button__CLOSE"]' with the `clickable` option.

Reply:
nice